index.html

```diff
@@ -0,0 +1,64 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <title>Document</title>
+</head>
+<body>
+    <header class="navbar">
+        <div class="welcome">
+            <a href="">Welcome</a>
+        </div>
+        <nav>
+            <img src="images/Union.png" alt="toolbar">
+        </nav>
+    </header>
+    <div class="item1">
+        <img src="images/frame.png" alt="item1">
+    </div>
+    <section class="middle">
+        <div>
+            <h1>Hey there. I'm Maximiliano.</h1>
+            <div class="item4">
+                <img src="images/frame5.png" alt="item4">
+            </div>
+            <h2>I'm a software developer</h2>
+        </div>
+        <div class="item3">
+            <img src="images/Frame3.png" alt="item2">
+        </div>
+        <p><strong>I can help you build a product , feature or website Look through some of my work and<br>experience! If you like what you see and have a project you need coded,<br>don’t hestiate to contact me.</strong></p>
+    </section>
+    <div>
+        <div class="item6">
+            <img src="images/Frame6.png" alt="item6">
+        </div>
+        <div class="item2">
+            <img src="images/Frame2.png" alt="item2">
+        </div>
+    </div>
```

Comment:
As requested by the previous reviewer to use the background image should be used as a `background-image` in the CSS where you can use the png background image from the Figma design here I have put the background image you should have to use. 👇 

<img width="375" alt="header-illsutration-mobile@2x" src="https://user-images.githubusercontent.com/34578631/170758703-b916e9d1-7a97-4381-abfa-5deb09ae1b54.png">

In this case, all the `img` have to be removed since this background image has all of them at once.